packages/create-sitecore-jss/src/common/processes/transform.ts

```diff
@@ -147,11 +147,42 @@ export const diffAndWriteFiles = async ({
   }
 };
 
+type TransformOptions = {
+  /**
+   * Determines whether a file should be copied
```

Comment:
Name of this option might be misleading. It's more a "is file for copy **only**". Maybe we just add that to the doc.
```suggestion
   * Determines whether a file should be copied only (not rendered through ejs)
```

Reply:
Done